### PR TITLE
ActionSheet: add Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: concepts/05 Widgets/ActionSheet/05 Customize Item Appearance.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ For a minor customization of **ActionSheet** buttons, you can define [specific f
         });
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ActionSheet/05 Customize Item Appearance.md:14]: We need make the same changes to this article as we did, for example, to the [TileView article](https://github.com/DevExpress/devextreme-documentation/pull/413/files#diff-9d7a06ab9bff92e89bc374ff40ad26d4R1).

File: concepts/05 Widgets/ActionSheet/10 Specify Display Mode.md [hunk 2]
```diff
@@ -35,6 +35,51 @@ By default, the **ActionSheet** comes up from the bottom of the page. If you set
         // ...
     })
 
+##### Vue
+
+    <template>
+        <DxActionSheet
+            :data-source="actionSheetData"
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ActionSheet/10 Specify Display Mode.md]:
```suggestion

```

File: concepts/05 Widgets/ActionSheet/10 Specify Display Mode.md [hunk 3]
```diff
@@ -35,6 +35,51 @@ By default, the **ActionSheet** comes up from the bottom of the page. If you set
         // ...
     })
 
+##### Vue
+
+    <template>
+        <DxActionSheet
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ActionSheet/10 Specify Display Mode.md]:
```suggestion
        <DxActionSheet ...
```

File: concepts/05 Widgets/ActionSheet/10 Specify Display Mode.md [hunk 4]
```diff
@@ -35,6 +35,51 @@ By default, the **ActionSheet** comes up from the bottom of the page. If you set
...
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { ActionSheet } from 'devextreme-react/action-sheet';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <ActionSheet
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ActionSheet/10 Specify Display Mode.md]:
```suggestion
                <ActionSheet ...
```

File: concepts/05 Widgets/ActionSheet/10 Specify Display Mode.md [hunk 5]
```diff
@@ -35,6 +35,51 @@ By default, the **ActionSheet** comes up from the bottom of the page. If you set
...
+                    dataSource={actionSheetData}
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/ActionSheet/10 Specify Display Mode.md]:
```suggestion

```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/ActionSheet/05 Customize Item Appearance.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/ActionSheet/05 Customize Item Appearance.md:14]: fixed

**Review by RomanTsukanov — APPROVED** [review]

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**